src/stores/spaces/SpaceStore.ts

```diff
@@ -1070,7 +1078,7 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
             this.emit(MetaSpace.Favourites);
         }
     }
-
+    // TODO on video room changed
```

Comment:
Looks like this should be done or more context provided along with an issue created

Reply:
I updated the comment. Dont know if its better to not have the comment at all?
Is there even a plan to make it possible for room types to change?

I think it is more realistic, that we move video rooms to not be room type but somthing similar to a widget layout.
Then a listener would need to be added to that state change.

Reply:
Room types are immutable. But not inherently atomic. There's no guarantee that the server will send you the create event in the very first sync containing a given room so it may change in your depiction of a synced room.